src/smf/packet-smf.c

```diff
@@ -881,17 +879,16 @@ static void smf_proto_add_base64_string(proto_tree *tree, packet_info *pinfo, in
     int offset, int size)
 {
     char* str;
-    gsize len = size;
    
     str = tvb_get_string_enc(NULL, tvb, offset, size, ENC_ASCII);
-    if (len > 1) {
+    if (size > 1) {
         if (strlen(str) > 1) {
+            gsize len = size; // This is for type conversion, g_base64_decode_inplace wants gsize for len
             g_base64_decode_inplace(str, &len);
```

Comment:
you probably do still need to null terminate the new str after calling g_base64_decode_inplace().  I assume the null termination code was added for some reason some time in the past.  It just needs to be here, in the if statement where the string is changed.